.github/workflows/test-multiple-builds.yml

```diff
@@ -69,7 +69,6 @@ jobs:
         run: |
           sed -i~ "s/<rootDir>\/src\(.*\)\.ts/<rootDir>\/dist\/umd\1.development.js/" package.json
           sed -i~ 's/"test:ci":.*,$/"test:ci": "jest",/' package.json
-          sed -i~ 's/= await import(/= require(/' tests/devtools.test.tsx
```

Comment:
should remove L80 too?

Reply:
Good call! [Done](https://github.com/pmndrs/zustand/pull/1753/commits/3befb2bdbbae1687faf492f3dad873594a5e6920)

Reply:
I didn't realize that one.
I meant this: `sed -i~ 's/= await import(/= require(/' tests/devtools.test.tsx`.

Reply:
Oh! I'm surprised that it did not cause the tests to fail this time 🤔 Fixed it in [this commit](https://github.com/pmndrs/zustand/pull/1753/commits/f5560adbdcce5e029dc87f34919d8038ebf6e435)